docs: adjust flow doc due to batch mode (#1677)
GreptimeTeam/docs: docs/user-guide/flow-computation/continuous-aggregation.md

```sql
INSERT INTO ngx_access_log VALUES
    ("cli1", 200, 100, now()),
    ("cli2", 200, 104, now()),
    ("cli3", 200, 120, now()),
    ("cli4", 200, 124, now()),
    ("cli5", 200, 140, now()),
    ("cli6", 404, 144, now()),
    ("cli7", 404, 160, now()),
    ("cli8", 404, 164, now()),
    ("cli9", 404, 180, now()),
    ("cli10", 404, 184, now());
```

> wait at least one second for flow to update results to sink table:

Reviewer: ```suggestion
wait a few seconds for flow to update results to sink table:
```
